ui/app/common/patient-search/controllers/patientsListController.js

```diff
@@ -2,8 +2,8 @@
 
 angular.module('bahmni.common.patientSearch')
 .controller('PatientsListController', ['$scope', '$window', 'patientService', '$rootScope', 'appService', 'spinner',
-    '$stateParams', '$bahmniCookieStore', 'printer', 'configurationService',
-    function ($scope, $window, patientService, $rootScope, appService, spinner, $stateParams, $bahmniCookieStore, printer, configurationService) {
+    '$stateParams', '$bahmniCookieStore', 'printer', 'configurationService', "$timeout",
```

Comment:
Do we need this change? I dont see the `$timeout` being used anywhere in the file

Reply:
`$timeout` is used for the case of targeted tabs. If we do not add this, we will encounter an undefined error message in the programs or any other list view while opening the targeted tab.